app/views/layouts/_nav.html.erb

```diff
@@ -11,17 +11,35 @@
     <!-- Right Nav Section -->
 
     <ul class="right">
+      <%= render "layouts/search_bar" %>
+
       <% if current_user %>
-        <% if current_user.try(:admin?) %>
-          <li> Admin</li>
-        <%- else-%>
-          <li><%= image_tag "#{current_user.profile_photo}", id: "profile_photo"%></li>
-        <%- end -%>
-      <%- end -%>
-      <%- if current_user -%>
-        <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+        <li class="has-dropdown">
+
+          <% if current_user.profile_photo.nil? %>
+            <a href="#"><%= image_tag "couch-potato.jpg", id: "profile_photo"%></a>
+          <%- else-%>
+            <a href="#"><%= image_tag "#{current_user.profile_photo}", id: "profile_photo"%></a>
+          <%- end -%>
```

Comment:
It would be nice if this logic was in a helper or in the model:

```
class User
  def display_photo
    profile_photo || path_to_couch_potato_image
  end
end
```